core/common/buffer_view.hpp

```diff
@@ -31,6 +31,10 @@ namespace kagome::common {
       return hex_lower(*this);
     }
 
+    std::string_view toStringView() const {
+      return {(const char *)data(), (size_t)size()};
+    }
+
```

Comment:
`byte2str` (include "common/bytestr.hpp") can convert any (vector/view/blob) to `string_view` (not only BufferView)